### DISCUSSION TITLE
Error of version of Github app example

### DISCUSSION THREAD

**Discussion by xavir2213** [community/community#108641] [Programming Help]
### Select Topic Area

Question

### Body

Hi i am trying to build an extension for my project of which is about creating an extension where people can check their[ KE Duplicate bill](https://printebill.com/k-electric-online-bill/) and i have downloaded the example file of extension from the github but when i was checking it was build on the v2 on v3 which is now necessary for Google chrome and other stores

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @xavir2213 ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions. 

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
